exasol-notes.md

```diff
@@ -0,0 +1,20 @@
+# Exasol Development Dev Notes
+
+
+## Development Environment
+
+All dependencies and tools required for development (except Python), according to `make prerequisites`, are provided by the Nix flake `flake.nix`.
+
+This environment can be activated by running the following command:
+
+```shell
+nix develop
+```
+
+Regarding Python, you are free to choose whatever fits your needs best. During development, Rye was used for providing and managing the Python version
+(`.python-version`). For development itself, a virtual Python environment was used. In any case, make sure you have an appropriate Python environment
+and have activated it so the path, etc., are set correctly, e.g., via:
+
+```shell
+source .venv/bin/activate.fish
```

Comment:
hi @Nicoretti thanks for the PR.

While you keep making progress on the connector itself, a couple of comments if I may:
1. we can add any specific developer notes [here](https://github.com/open-metadata/OpenMetadata/tree/main/openmetadata-docs/content/v1.5.x-SNAPSHOT/developers/contribute). Docs are md files in this same repo, so any specific consideration needed for exasol we can add it there
2. I'd avoid introducing - at least in this same PR - athe `flake.nix` and `flake.lock` or any new env management tool. This is something that would live in the whole project, and might require a different/extended discussion.

Thanks

Reply:
Hi @pmbrull,

Feel free to provide any feedback at any time, it is highly appreciated!

Sorry, it seems I haven't communicated my intentions well enough in the PR description,
so let me try to clarify your points from my perspective.

1. Thanks for pointing me to the exact location. I'll make sure the final and official documentation for the connector will end up there. For now, these are merely some notes for me or my colleagues, in case they need to pick up on the work. (Relevant pieces will be migrated to the place you mentioned, and irrelevant pieces will be purged for the "final" PR)

2. I wouldn't dare to just introduce a new complex tool out of the blue, especially `nix`, due to my kind of "love & hate" relationship with it :sweat_smile:. That said, it helps me keep my system clean and bootstrap a reproducible setup quickly. Due to the nature of `nix` flakes, they need to be in the git index to work (so I committed it for now). I'll make sure that once I remove the draft status of the PR, the `nix` files won't be part of it anymore.

:spiral_notepad:  Side Note: August is quite packed for me; therefore, it is likely that getting the PR out of draft status will happen earliest in September.

Hope that helps to clarify the details a bit better.

Best,
Nico